tests/test_main.py

```diff
@@ -22,87 +32,115 @@ def created_person_list(people_data):
-           created_person_list[0].wife.husband == created_person_list[0], (
+def test_create_person_list_has_wife_and_wife_have_husband(
+    people_data, created_person_list
+):
+    assert (
+        hasattr(created_person_list[0], "wife")
+        and created_person_list[0].wife.husband == created_person_list[0]
+    ), (
         f"Person with 'name' {created_person_list[0].name} should have "
         f"attribute 'wife' with name {created_person_list[0].wife.name} and "
         f"Person.wife.husband should links to that Person"
     )
 
 
 def test_create_person_list_has_no_wife(people_data, created_person_list):
-    assert hasattr(created_person_list[1], 'wife') is False, (
+    assert hasattr(created_person_list[1], "wife") is False, (
         f"Person with 'name' {created_person_list[1].name} should not have "
         f"attribute wife"
     )
 
 
 def test_person_class_attribute_people(people_data, created_person_list):
-    assert all(isinstance(person, Person) for person in Person.people.values()), (
-        "All elements of Person class attribute 'people' should be Person instances"
-    )
-    assert len(Person.people) == len(people_data), (
-        "Length of Person class attribute people should be equal to length of initial list"
+    assert all(
+        isinstance(person, Person) for person in Person.people.values()
+    ), "All elements of Person class attribute 'people' should be Person instances"
+    assert len(Person.people) == len(
+        people_data
+    ), "Length of Person class attribute people should be equal to length of initial list"
+
+
+def test_create_person_list_returns_only_entering_people(
+    people_data, created_person_list
+):
+    assert len(people_data) == len(
+        created_person_list
+    ), "Length of passed list should equal to length of returned list"
+    assert [person["name"] for person in people_data] == [
+        person.name for person in created_person_list
+    ], (
+        "People, that are passed to the function should equal to people, "
+        "that are returned"
     )
 
 
-def test_create_person_list_returns_only_entering_people(people_data, created_person_list):
-    assert len(people_data) == len(created_person_list), (
-        "Length of passed list should equal to length of returned list"
-    )
-    assert [person['name'] for person in people_data] == [person.name for person in created_person_list], (
-        "People, that are passed to the function should equal to people, "
-        "that are returned"
-    )
\ No newline at end of file
+def test_person_instance_attribute_wife_and_husband_doesnt_exists():
+    with open(path_to_main()) as file:
+        tree = ast.parse(file.read())
+
+    assert (
+        len(
+            tree.__dict__["body"][0]
+            .__dict__["body"][1]
+            .__dict__["args"]
+            .__dict__["args"]
+        )
+        == 3
+    ), "'__init__' should takes only two arguments 'name' and 'age'!"
```

Comment:
The `test_person_instance_attribute_wife_and_husband_doesnt_exists` test checks the number of arguments in the `__init__` method by parsing the AST. This approach is fragile and may break if the structure of the `__init__` method changes. Consider testing the functionality directly rather than relying on the method's structure.